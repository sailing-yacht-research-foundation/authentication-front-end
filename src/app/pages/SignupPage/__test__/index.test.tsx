import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { SignupPage } from '..';

const shallowRenderer = createRenderer();

describe('<SignupPage />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<SignupPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});