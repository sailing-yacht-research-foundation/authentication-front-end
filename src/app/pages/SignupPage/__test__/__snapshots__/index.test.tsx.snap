// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignupPage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <styled.div />
  <Row
    align="middle"
    justify="center"
    style={
      Object {
        "height": "calc(100vh - 100px)",
        "marginTop": "100px",
        "padding": "0 15px",
      }
    }
  >
    <SignupForm />
  </Row>
</React.Fragment>
`;
